books/tour/src/morphology.md

```diff
@@ -35,9 +35,9 @@ other hand, roots always follow either the pattern __VC__, __CVC__, __CCV__
 or __CCVC__.
 <!--In particular, they always have exactly two consonants and one or more vowels and/or apostrophes.-->
 
-Particles and roots starting with a vowel, **l, n** or **r** are technically preceded by a
+Particles and roots starting with a vowel or coda (**l, n** or **r**) are technically preceded by a
```

Comment:
Technically "coda" is a term reserved for consonants consonants at the end of a syllable. These consonants should not be called coda outside of this situation. We should find a better name for them.